lms/static/sass/course/modules/_calculator.scss

```diff
@@ -62,6 +68,12 @@ div.calc-main {
       @include box-sizing(border-box);
       padding: lh();
 
+      label.sr {
```

Comment:
If we need to keep this, can we give this element another class (like ``help-label``) so we can keep the rules scoped, but tied to a class rather than an element?